buildtest/menu/show.py

```diff
@@ -1,28 +1,38 @@
 import json
-import sys
+import os
 
-from buildtest.defaults import supported_schemas
 from buildtest.buildsystem.schemas.utils import (
     load_schema,
     get_schemas_available,
     get_schema_fullpath,
+    here,
 )
 
 
 def show_schema_layout(args):
-    """Implements method ``buildtest show schema``"""
-
-    # buildtest show schema --name script
-    if args.name is None:
-        sys.exit("Please provide the name of a schema (e.g., script) to show.")
-
-    # Must be supported
-    if args.name not in supported_schemas:
-        sys.exit(
-            "%s is not a supported schema. Options include %s"
-            % (args.name, "\n".join(supported_schemas))
-        )
-
+    """This method implements command ``buildtest show schema`` which displays json schemas
+       supported by buildtest. The input ``args`` is an instance of argparse class that contains
+       user selection via command line. The method can display the global schema when
+       ``buildtest show schema --global`` is specified or any name schema via
+       ``buildtest show schema --name <NAME>``. The result is an output of the json schema on the console.
+
+       Parameters:
+
+       :param args: instance of argparse class
+       :type args: <class 'argparse.Namespace'>
+       :result: output of json schema on console
+    """
+
+    # implements ``buildtest show schema --global``. Since we can't use args.global we set this to main
```

Comment:
Same here, args._global is clear.

Reply:
just to be clear you want to change ``dest=name`` to ``dest=_global`` to resemble the name of the option. I couldn't understand is ``args.global`` a reserved attribute in argparse